terraform/central-account/terraform.tfvars.example

```diff
@@ -27,6 +29,7 @@ lb-authentication-authorization-endpoint = "https://dev-12345.okta.com/oauth2/de
 lb-authentication-client-id = "<the-client-id-of-the-app>"
 lb-authentication-client-secret = "<the-client-secret-of-the-app>"
 lb-authentication-issuer = "https://dev-12345.okta.com/oauth2/default"
+lb-authentication-jwt-email-key = "sub"
```

Comment:
This should also be `email` (Okta and Google are both email), so let's use that for now - I will test with more IdPs in the future to confirm.